### PR TITLE
Fixes a bug in livesample

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Fixes `ReferenceError: filesArray is not defined` error.

### CODE REVIEW

File: files/en-us/web/api/navigator/share/index.md [hunk 1]
```diff
@@ -153,14 +155,14 @@ const shareData = {
...
 ```
 
-### Result
+#### Result
 
 Click the button to launch the share dialog on your platform.
 Text will appear below the button to indicate whether the share was successful or provide an error code.
 
-{{EmbedLiveSample('Examples')}}
+{{EmbedLiveSample('sharing_a_url')}}
```
Comment (wbamberg) [inline files/en-us/web/api/navigator/share/index.md:163]: Just FYI, you can just pass the heading text here, like "Sharing a URL". Both work, but personally I like that way.

**Review by wbamberg — COMMENTED** [review]
This looks fine, but the live sample still doesn't work, I get:

> Error: TypeError: navigator.share is not a function

Firefox and Chrome. Do you?

**Comment by OnkarRuikar** [comment]
It works for me on Chrome 102.
As per the compatibility table Firefox doesn't support the API. Chromium started supporting it from v93.

**Comment by wbamberg** [comment]
> It works for me on Chrome 102. As per the compatibility table Firefox doesn't support the API. Chromium started supporting it from v93.

Yeah, I'm on 102 as well. 🤷

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Comment by OnkarRuikar** [comment]
Chrome:

https://user-images.githubusercontent.com/87750369/172990012-9e008e99-0dbe-48fb-be58-3528b2cfbfb0.mov

**Comment by wbamberg** [comment]
Oh, I believe you, I just don't know why it doesn't work for me. But as long as it works for some (most?) people I'm happy with it.

**Pull request merged**